spec/requests/api/v1/users_request_spec.rb

```diff
@@ -164,6 +164,26 @@
         it { expect(user.reload.last_name).to eq(last_name) }
       end
 
+      response 200, "updates a user frozen by franceconnect", document: false do
+        let(:user) do
+          create(:user,
+                 first_name: "Jean",
+                 organisations: [organisation],
+                 logged_once_with_franceconnect: true)
+        end
+
+        let(:first_name) { "Alain" }
+        let(:address) { "10 rue du Havre, Paris" }
+
+        schema "$ref" => "#/components/schemas/user_with_root"
+
+        run_test!
+
+        it { expect(user.reload.first_name).not_to eq(first_name) }
+
+        it { expect(user.reload.address).to eq(address) }
```

Comment:
on peut même faire un `expect(user.reload).to have_attributes(...)` pour grouper ça en un seul it